SC Evo Complete/SCEvo_Core.SC2Mod/Base.SC2Data/GameData/EffectData.xml

```diff
@@ -450,41 +425,40 @@
         <PeriodicOffsetArray value="-0.4687,0.2812,0"/>
         <PeriodicOffsetArray value="0.5937,-1.5625,0"/>
         <PeriodicOffsetArray value="0.5312,-0.25,0"/>
+        <EffectRandMode value="Shuffle"/>
+        <OffsetRandMode value="Shuffle"/>
+        <PeriodRandMode value="Shuffle"/>
     </CEffectCreatePersistent>
-    <CEffectCreatePersistent id="ValkyrieSCBWWeapon@CP">
-        <EditorCategories value="Race:Terran"/>
-        <WhichLocation Value="TargetUnit"/>
-        <PeriodCount value="4"/>
-        <PeriodicEffectArray value="ValkyrieSCBWWeapon@Set"/>
-        <PeriodicPeriodArray value="0"/>
-        <PeriodicPeriodArray value="0.65"/>
-        <PeriodicPeriodArray value="0.65"/>
-        <PeriodicPeriodArray value="0.65"/>
-    </CEffectCreatePersistent>
-    <CEffectDamage id="ValkyrieSCBWWeapon@Damage" parent="DU_WEAP_MISSILE">
+    <CEffectDamage id="ValkyrieSCBWWeapon@Damage" parent="ExplosiveBaseDamageSCBW">
         <EditorCategories value="Race:Terran"/>
+        <AttributeBonus index="Light" value="-1.5"/>
+        <AttributeBonus index="Armored" value="1.5"/>
+        <ShieldBonus value="1.5"/>
+        <Kind value="Splash"/>
         <Amount value="4.5"/>
-        <AreaArray Radius="1.5" Fraction="1"/>
-        <AreaArray Radius="2" Fraction="0.5"/>
-        <AreaArray Radius="2.5" Fraction="0.25"/>
+        <AreaArray Radius="0.1562" Fraction="1"/>
+        <AreaArray Radius="1.5625" Fraction="0.5"/>
+        <AreaArray Radius="3.125" Fraction="0.25"/>
+        <IncludeArray Effect="ValkyrieSCBWWeapon@CP" Value="Target"/>
         <SearchFilters value="Air,Visible;Player,Ally,Neutral,Missile,Stasis,Dead,Invulnerable,Benign,Passive"/>
-        <AttributeFactor index="Light" value="-0.3334"/>
-        <AttributeFactor index="Armored" value="0.3334"/>
-        <ShieldFactor value="0.3334"/>
     </CEffectDamage>
     <CEffectLaunchMissile id="ValkyrieSCBWWeapon@LM">
         <EditorCategories value="Race:Terran"/>
         <ImpactEffect value="ValkyrieSCBWWeapon@Damage"/>
         <AmmoUnit value="ValkyrieSCBWWeapon@Missile"/>
+        <LaunchLocation Value="CasterUnit"/>
+        <ImpactLocation Value="TargetPoint"/>
```

Comment:
the key to making the valkyrie missiles spread out is to target a point instead of a unit, otherwise it won't know what to fire at since the offsets aren't units. then since it's splash damage anyway we don't need search effects